File: conditions/time.py
import datetime

currenttime = datetime.datetime.now()

print(currenttime.hour)

hour = 13

if hour < 12:
    print("Good Morning! Early bird gets the worm :) ")
elif hour < 15:
    print("Good Afternoon!")
else: 
    print("Good Good Evening! Don't go to sleep too late..")

File: conditions/time.c
#include <stdio.h>
#include <time.h>

int hour; 

int main()
{
   time_t now;
   struct tm*now_tm;

   time(NULL);
   now_tm = localtime (&now);
   hour = (now_tm->tm_hour);
   printf("The hour in military time is: %d\n", hour);
   printf("Tell me the hour in military time:\n");
   scanf("%d", &hour);
   if(hour < 12){
    printf("Good Morning! Early bird gets the worm :)\n");
   }else if (hour < 18){
    printf("Good Afternoon!\n");
   }else{
    printf("Good Evening!\n");
   }
    return 0;
}